plugins/readtimer.koplugin/main.lua

```diff
@@ -15,6 +23,9 @@ local ReadTimer = WidgetContainer:extend{
 }
 
 function ReadTimer:init()
+    self.show_timer_value_in_header = G_reader_settings:readSetting("show_timer_value_in_header")
+    self.show_timer_value_in_footer = G_reader_settings:readSetting("show_timer_value_in_footer")
```

Comment:
For a plugin settings, I would rather have them all use an identical prefix, ie. `readtimer_show_value_in_header`.